### DISCUSSION TITLE
How do I update/remove collision detection?

### DISCUSSION THREAD

**Discussion by Tzikas** [enable3d/enable3d#266] [Help / Questions] [answered]
I have a car patrolling a road similar to this https://enable3d.io/examples/enemy-patrols-on-platform.html?_sm_nck=1 

I want to increase the scale of the road and have the patrol radius also be updated, but even when the road increases in scale the car still collides with the previous  road dimensions.  

      //This is called once when the car is created 
      this.third.physics.add.collider(sensor, this.road, event => {
        if (event === 'end') car.body.setAngularVelocityY(angVelY)
        else car.body.setAngularVelocityY(0)
      })
      
      
      //This is how i update the size of the road
      this.road.scale.x++
      this.road.scale.y++
      
      How do I update/remove collision detection?

**Comment by yandeu** [comment, marked as answer]
You can't change the size of a physics body. You have to remove it and make a new one.
